<?xml version="1.0"?>
<launch>
    <node pkg="depthimage_creater" type="range_image_projection" name="range_image_projection" output="screen">
        <remap from="/cloud"        to="/camera/depth_registered/points/tf" />
        <remap from="image"         to="/camera/color/image_raw" />
        <remap from="/camera_info"  to="/camera/color/camera_info" />
    </node>
    
    <!--Transform PointCloud-->
	<node pkg="sensor_fusion" type="laser_transform_pointcloud" name="realsense_transform">
		<param name="target_frame" type="string" value="camera_link" />
        <remap from="/cloud"    to="/camera/depth_registered/points" />
        <remap from="/cloud/tf" to="/camera/depth_registered/points/tf" />
	</node>


</launch>
